src/lsm/manifest_level.zig

```diff
@@ -197,17 +321,17 @@ pub fn ManifestLevelType(
             visibility: Visibility,
             snapshots: []const u64,
             direction: Direction,
-            key_range: ?KeyRange,
+            range: ?KeyRange,
         ) Iterator {
             for (snapshots) |snapshot| {
                 assert(snapshot <= lsm.snapshot_latest);
             }
 
             const inner = blk: {
-                if (key_range) |range| {
-                    assert(compare_keys(range.key_min, range.key_max) != .gt);
+                if (range) |range_| {
```

Comment:
We try hard never to use `_`.

In this case, the `key_range` name works better I think.

Reply:
I had to change these because we now have a function called [key_range](https://github.com/tigerbeetle/tigerbeetle/blob/17f09e23c370e7a3ea70f0ed080b8e03304ffc0e/src/lsm/manifest_level.zig#L192) and these variable names were conflicting with that!

Let me think about a better naming scheme, with `LevelKeyRange` coming into the picture and a `key_range` inside it as well, there are a lot of `key_range` instances flying around. :)

Reply:
FYI, I was able get around this, I reverted all variables names to what they were before as part of https://github.com/tigerbeetle/tigerbeetle/pull/944/commits/5b58bf6ad3eeea6fbd9ba201e7369a4d32edd14d.